src/components/Notifier.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+
+let openSnackbarFn;
+class Notifier extends React.Component {
+    state = {
+        open: false,
+        message: '',
+    };
+
+    openSnackbar = ({ message }) => {
+        this.setState({
+            open: true,
+            message,
+        });
+    };
+
+    handleSnackbarClose = () => {
+        this.setState({
+            open: false,
+            message: '',
+        });
+    };
+
+    componentDidMount() {
+        openSnackbarFn = this.openSnackbar;
+    }
+
+    render() {
+        const message = (
+            <span
+                id="snackbar-message-id"
+                dangerouslySetInnerHTML={{ __html: this.state.message }}
+            />
+        );
+
+        return (
+            <Snackbar
+                anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+                message={message}
+                autoHideDuration={3000}
+                onClose={this.handleSnackbarClose}
+                open={this.state.open}
```

Comment:
Wäre es sinnvoll, noch eine _action_ mit einem Schliessen-Button hinzuzufügen?
```suggestion
                open={this.state.open}
                action={[
                    <IconButton
                        key="close"
                        onClick={this.handleSnackbarClose}
                    >
                        <CloseIcon />
                    </IconButton>
                ]}
```

Reply:
Die Meldung verschwindet automatisch nach 3s, zudem geht es auch weg wenn man irgendwo hinklickt, meinst du nicht das dies reicht :) ?

Reply:
Die Meldung verschwindet automatisch nach 3s, zudem geht es auch weg wenn man irgendwo hinklickt, meinst du nicht das dies reicht :) ?

Reply:
Klar, das andere wäre ein kleiner Pluspunkt, welchen man auch weglassen kann. Kein Problem